### PR TITLE
Upgrade to JDBI 2.62

### PR DESCRIPTION
**Pull request by arteam**
[Release notes](https://github.com/jdbi/jdbi/blob/master/RELEASE_NOTES)

JDBI 2.62 has several useful features such as:

* Positional bindings in SqlObject API (incidentally contributed by me)
* Ability to bind arguments in SqlObject API as Map
* Better Java 8 support

### CODE REVIEW

File: dropwizard-jdbi/src/test/java/io/dropwizard/jdbi/timestamps/DateTimeSqlTimestampTest.java [hunk 1]
```diff
@@ -101,11 +101,10 @@ public void testReadTimestamp() {
 
     public interface FlightDao {
 
-        @SqlUpdate("INSERT INTO flights(flight_id, departure_airport, arrival_airport,departure_time, arrival_time) " +
-                "VALUES (:flight_id, :departure_airport, :arrival_airport, :departure_time, :arrival_time)")
-        int insert(@Bind("flight_id") String flightId, @Bind("departure_airport") String departureAirport,
-                   @Bind("arrival_airport") String arrivalAirport,
-                   @Bind("departure_time") DateTime departureTime, @Bind("arrival_time") DateTime arrivalTime);
+        @SqlUpdate("INSERT INTO flights(flight_id, departure_airport, arrival_airport, departure_time, arrival_time) " +
+                "VALUES (:0, :1, :2, :3, :4)")
```
Comment (joschi) [inline dropwizard-jdbi/src/test/java/io/dropwizard/jdbi/timestamps/DateTimeSqlTimestampTest.java]: Is this change really necessary? I think named parameters are way less error-prone than mere numbered parameters in this case.

Comment (arteam) [inline dropwizard-jdbi/src/test/java/io/dropwizard/jdbi/timestamps/DateTimeSqlTimestampTest.java]: Fair enough. 
That's why I moved this change into a separate commit.
